src/GameMachine.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class GameMachine {
+
+    private List<Game> games = List.of(
+            new Game("Warcraft III", 60),
+            new Game("Diablo II", 80),
+            new Game("Starcraft", 35)
+    );
+
+    public List<Game> getGames() {
+        return games;
+    }
+
+    public void setGames(List<Game> games) {
+        this.games = games;
+    }
+
+    public Game getGame(String gameName) throws NotSuchGameException {
```

Comment:
Exception should be NoSuchGameException, but you could also use NoSuchElementException which I think is part of Java (or maybe Spring. I don't rememeber) but change the exception name

Reply:
Yes, but int the task description was written that I should make my own class Exception.